### PR TITLE
Adding new rook commands file

### LINKED ISSUE #708: [doc] 5  Commands and Command Prompts Used in This Guide
### Type of report
   - [x] bug
   - [ ] feature request

### Source Document
5  Commands and Command Prompts Used in This Guide:

https://susedoc.github.io/doc-ses/master/single-html/ses-rook/#id-1.7.2.13

### Summary
Does it make sense to have the `5.1 Salt Related Commands` and `5.2 Ceph Related Commands` in the rook guide? 
When deploying with rook, there is no salt involved. Also `cephadm` is not there afaiu.

Instead, there is the toolbox container

### PR DESCRIPTION
**Pull request by asettle**
1. Removed Ceph-specific content for this guide
2. Adds new file for rook prompts

Closes #708

### CODE REVIEW

**Comment by asettle** [comment]
This replaces https://susedoc.github.io/doc-ses/master/single-html/ses-rook/#id-1.7.2.13 with a more Rook-specific variant. Any further suggestions on commands or examples - @mgfritch @ml8mr ?

File: xml/rook_prompts.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+</info>
+
+ <para>
+  As a &rook; cluster administrator, you will be configuring and adjusting the
+  cluster behavior by running specific commands. There are several types of
+  commands you will need:
+ </para>
+
+ <sect2>
+  <title>Helm Related Commands</title>
```
Comment (lproven) [inline xml/rook_prompts.xml:25]:
```suggestion
  <title>Helm-Related Commands</title>
```

File: xml/rook_prompts.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  <para>
+   For example:
+  </para>
+<screen>
+&prompt.root;helm3 chart pull
+</screen>
+ </sect2>
+
+ <sect2>
+  <title>&kube; Related Commands</title>
```
Comment (lproven) [inline xml/rook_prompts.xml:43]:
```suggestion
  <title>&kube;-Related Commands</title>
```

**Review by lproven — CHANGES_REQUESTED** [review]
Trivial, just 2 hyphens.

**Review by lproven — APPROVED** [review]

**Review by mgfritch — APPROVED** [review]

**Pull request merged**